docs/doc-draft.md

```diff
@@ -23,24 +23,26 @@ This is diagram as given in #14, which is not actually how this works, but helpf
 sequenceDiagram
 participant U as User
 participant D as Docker
-participant YP as Youki(Parent Process)
-participant YI as Youki(Init Process)
+participant Y_Main as Youki(Main Process)
+participant Y_Intermediate as Youki(Intermeidate Process)
```

Comment:
I've been thinking about the name a little bit, how about `Joint Process`? This is quite difficult and I'm not sure about it, so I'd like to hear your opinion.

Reply:
I think `intermediate` is likely the closest description to what the middle/first fork/intermediate process does, tbh. I think init process is probably the right name for the second/init process. I think `main` can be used to describe the main process.

I think the parent/child process was confusing because Youki main process and intermediate process can be parent/child, but also intermediate and init process can be parent/child. So we shouldn't call the `first fork` the child process. This is part of the reason why I refactored the parent and child process structure to just use a channel.

I know this is hard LOL. What is your thinking behind `Joint Process`?

Reply:
I wanted to emphasize that it is a process of highlighting and joining main and init. This is honestly difficult to name. I think `intermediate` is at least a better name than before, so let's go with that for now.